02_assignments/assignment.md

```diff
@@ -48,3 +54,4 @@ Your answer here...
 
 
 If you encounter any difficulties or have questions, please don't hesitate to reach out to our team via our Slack at `#cohort-3-help`. Our Technical Facilitators and Learning Support staff are here to help you navigate any challenges.
+[Script.txt](https://github.com/YMakushenko/shell/files/15125627/Script.txt)
```

Comment:
Just in case the "Script.txt" doesn't open, I'm copying my script here:
git credential-manager github login
git clone https://github.com/YMakushenko/shell
cd 02_assignments/clues/food/cake
cat *cake.txt
cd ../../..
touch SecretPassword.sh
echo "Paper Rings" > SecretPassword.sh
cd clues/shows/friends
find season_*/ep_18.txt
find season_*/ep_19.txt
echo "10" >> ../../../SecretPassword.sh
cd season_6
awk '{print $5}' ep_21.txt
echo "Meets" >> ../../../../SecretPassword.sh
cd ../../..
cd movies/space_wars/
awk '{print $5}' fifth_movie.txt
echo "and" >> ../../../SecretPassword.sh
cd ../..
cd albums/red/
cat song_*.txt
awk '{print $3}' song_5.txt
echo "Lucky" >> ../../../SecretPassword.sh
cd ../..
cd movies/hanger_games/
awk '{print $4}' movie_4.txt
echo "the" >> ../../../SecretPassword.sh